spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-db2.sql

```diff
@@ -18,7 +18,7 @@ CREATE TABLE INT_GROUP_TO_MESSAGE  (
 CREATE TABLE INT_MESSAGE_GROUP  (
 	GROUP_KEY CHAR(36) NOT NULL,
 	REGION VARCHAR(100) NOT NULL,
-	MARKED BIGINT,
+    CONDITION VARCHAR(255),
```

Comment:
We'll need a migration guide entry for the schema change.

Reply:
Sure! Will address it after merge